File: Foundation/Scratch/Getting Started/c++/l002ReverseNumberReturnType.cpp
#include <iostream>
using namespace std;
int reverseNumber(int n){
    int ans=0,lastDigit=0;
    while(n){
        lastDigit = n%10;
        n /= 10;
         
       ans = ans * 10 + lastDigit;
       
    }
    return ans;
    
}

int main(int argc, char **argv){
    int n;
    cin >> n;
cout<<reverseNumber(n);
    //write your code here
    return 0;
}

File: Foundation/Scratch/Getting Started/java/l002ReverseNumberReturnType.java
import java.util.*;

public class Main {
    public static Scanner scn = new Scanner(System.in);
public static int reverseNumber(int n){
    int ans=0,lastDigit=0;
    while(n!=0){
        lastDigit = n%10;
        n /= 10;
         
       ans = ans * 10 + lastDigit;
       
    }
    return ans;
    
}

public static void main(String[] args){
    int n = scn.nextInt();
   
System.out.println(reverseNumber(n));
    //write your code here
 
}
}
